### ISSUE TITLE
NotFoundHttpException not handled with admin URI

### ISSUE THREAD

**Issue by mohdfaizdev** [bagisto/bagisto#514]
![issue1](https://user-images.githubusercontent.com/22182116/52101163-6356d000-2600-11e9-8e1b-75901d776b83.png)
![issue2](https://user-images.githubusercontent.com/22182116/52101167-6782ed80-2600-11e9-8b90-c7b70499bfc8.png)

**Comment by rahulshukla-webkul** [comment]
@mohdfaizdev thanks for letting us know, we highly appreciate your effort. 

We make separate error pages for admin because we want to show error on admin section so that user can easily navigate. Only problem was we did not handle exception for anonymous user & included same admin layouts (eg. @extends('admin::layouts.content') ) that's why error is coming. Now we have solved this issue by using shop layout for anonymous users.

Once again, we thanks to you & looking for your contribution in our project.
